src/prefect/core/task.py

```diff
@@ -34,6 +34,21 @@
 VAR_KEYWORD = inspect.Parameter.VAR_KEYWORD
 
 
+# A sentinel value indicating no default was provided
+class NoDefault:
+    __slots__ = ()
+
+
+class SerializableNoDefault(NoDefault):
+    __slots__ = ()
+
+    def __repr__(self) -> str:
+        return "no_default"
+
+    def __reduce__(self) -> str:
+        return "no_default"
```

Comment:
Apologies, looks like we both tried to fix the mypy issue (fixed in #3490), so you now have a few merge conflicts.

Should be able to reuse the `NoDefault` class from `prefect.core.parameter` in your changes in `prefect.tasks.core.operators`, or just duplicate the code over there for a new sentinel. IMO there's no harm in duplicating that pattern in a few places - it's not user facing and there's a mild mental cost to indirection.